release_announcement

```diff
@@ -15,7 +15,7 @@ fi
 echo >> "$ANNOUNCEMENT"
 
 if [[ $PROJECT == foreman ]] ; then
-	# TODO: inform the user input is expected?
+	echo Enter the announcement text, end by CTRL+D:
```

Comment:
Technically if there already was something on stdin then it's not expected, but we can ignore that.

More important thing to consider: `read -p` prints on stderr so you can copy stdout (for example, using `wl-copy`). I'd like to maintain that option.

Reply:
`>&2`